dev-docs/bidders/bidscube.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: bidder
+title: BidsCube
+description: Prebid BidsCube Bidder Adaptor
+pbs: true
+pbjs: true
+biddercode: bidscube
+media_types: banner, video, native
+gdpr_supported: true
+pbs_app_supported: true
+---
+
+### Note:
+
+The Example Bidding adapter requires setup before beginning. Please contact us at support@bidscube.com
```

Comment:
Are you sure you want this to say "The Example Bidding adapter"? Maybe you meant to replace this with BidsCube?

Reply:
Yes, fixed